lib/go-tc/enum.go

```diff
@@ -595,6 +595,11 @@ func DSTypeFromString(s string) DSType {
 	}
 }
 
+//IsAnyMap returns weather the DSType is an ANY_MAP service.
```

Comment:
sp: weather -> whether, and I think godoc likes a space after the `//`

Reply:
Fixed spelling error and added white space after //